packages/ckeditor5-image/tests/imageresize/imageresizeui.js

```diff
@@ -82,29 +88,28 @@ describe( 'ImageResizeUI', () => {
 		} );
 	} );
 
+	it( 'should create `_resizeUnit`', () => {
+		const unit = plugin._resizeUnit;
+
+		expect( unit ).to.equal( '%' );
+	} );
+
 	describe( 'init()', () => {
```

Comment:
I'm confused by these `init()` tests:

*   "should have set "%" resize unit if not defined" asserts the value of the **command** which has nothing to do with `init()` or the `ImageResizeUI` plugin.
*   "should have set "px" resize unit" same as above, what does `expect( command.value.width.includes( 'px' ) ).to.be.true;` have to do with `ImageResizeUI`?

Shouldn't these tests be somewhere among `ImageResizeCommand` tests?

At the same time:

*   "should be disabled when command is disabled" should be under `init()`